### PR TITLE
Update php-cs-fixer and its config and fix the codebase

### PR DESCRIPTION
**Pull request by lyrixx**

### CODE REVIEW

File: src/Controller/ZoomController.php [hunk 1]
```diff
@@ -77,8 +77,9 @@ public function authenticate(Request $request, ZoomApplication $zoomApplication)
             $session->set(ZoomApplication::SESSION_KEY_STATE, $provider->getState());
 
             return new RedirectResponse($authUrl);
-        // Check given state against previously stored one to mitigate CSRF attack
-        } elseif (empty($request->query->get('state')) || $request->query->get('state') !== $session->get(ZoomApplication::SESSION_KEY_STATE)) {
+            // Check given state against previously stored one to mitigate CSRF attack
```
Comment (pyrech) [inline src/Controller/ZoomController.php:80]: I guess this comment should be moved above the following if

File: src/Zoom/MessageSender.php [hunk 2]
```diff
@@ -18,8 +18,8 @@
 
 class MessageSender
 {
-    const SANTA_EMOJI = "\xF0\x9F\x8E\x85";
-    const GIFT_EMOJI = "\xF0\x9F\x8E\x81";
+    public const SANTA_EMOJI = "\xF0\x9F\x8E\x85";
+    public const GIFT_EMOJI = "\xF0\x9F\x8E\x81";
```
Comment (pyrech) [inline src/Zoom/MessageSender.php:22]: those constants can be made private IIRC

File: src/Discord/ApiHelper.php [hunk 3]
```diff
@@ -18,10 +18,10 @@
 
 class ApiHelper
 {
-    const TOKEN_TYPE_BOT = 'Bot';
+    public const TOKEN_TYPE_BOT = 'Bot';
```
Comment (pyrech) [inline src/Discord/ApiHelper.php:21]: constant can be private

**Review by pyrech — CHANGES_REQUESTED** [review]

**Comment by lyrixx** [comment]
@pyrech thanks for the review, I addressed your comments.

**Review by pyrech — APPROVED** [review]

**Pull request merged**